SpywareFilter/sections/general_elemhide.txt

```diff
@@ -1,5 +1,9 @@
 !
-! Hide counters images
+! This section contains generic element hiding rules that block counters images, leftovers.
```

Comment:
`counters images` blocked by url. Leftovers only. It makes no sense to hide counters that are not blocked.